setup.py

```diff
@@ -54,7 +54,7 @@ def _get_test_env():
 
     def run(self):
 
-        cmd = ['py.test']
+        cmd = ['py.test -vv']
```

Comment:
We shouldn't merge this change in. For testing you can always put these things in `setup.cfg`.